Initial PTP documentation for ATIP (#542)
suse-edge/suse-edge.github.io: asciidoc/quickstart/eib.adoc

> It's also possible to add additional users, create the home directories, set user-id's, add ssh-key authentication, and modify group information. Please refer to the {link-eib-building-images}[upstream building images guide] for further examples.

Reviewer: ```suggestion
It is also possible to add additional users, create the home directories, set user-ids, add ssh-key authentication, and modify group information. Please refer to the {link-eib-building-images}[upstream building images guide] for further examples.
```

====

[#eib-configuring-rpm-packages]
=== Configuring RPM packages

> One of the major features of EIB is to provide a mechanism to add additional software packages to the image, so when the installation completes the system is able to leverage the installed packages right away. EIB permits users to specify the following:

Reviewer: ```suggestion
One of the major features of EIB is to provide a mechanism to add additional software packages to the image, so when the installation completes, the system can leverage the installed packages right away. EIB permits users to specify the following:
```

Please refer to the <<components-nmc, Edge Networking component>> for a more comprehensive explanation of the above configuration and examples of this feature.
====

[#eib-how-to-build-image]
== Building the image

> Now that we've got a base image and an image definition for EIB to consume, let's go ahead and build the image. For this, we simply use `podman` to call the EIB container with the "build" command, specifying the definition file:

Reviewer: ```suggestion
Now that we have a base image and an image definition for EIB to consume, let us proceed and build the image. For this, we simply use `podman` to call the EIB container with the "build" command, specifying the definition file:
```
